Source/Widgets/Geocoder/GeocoderViewModel.js

```diff
@@ -331,14 +341,79 @@ define([
     }
 
     function updateCamera(viewModel, destination) {
-        viewModel._scene.camera.flyTo({
-            destination : destination,
-            complete: function() {
-                viewModel._complete.raiseEvent();
-            },
-            duration : viewModel._flightDuration,
-            endTransform : Matrix4.IDENTITY
-        });
+        var scene = viewModel._scene;
+        var globe = scene.globe;
+        var ellipsoid;
+        if (defined(globe)) {
+            ellipsoid = globe.ellipsoid;
+        } else {
+            ellipsoid = scene.mapProjection.ellipsoid;
+        }
+
+        var camera = scene.camera;
+        var terrainProvider = scene.terrainProvider;
+        var availability = defined(terrainProvider) ? terrainProvider.availability : undefined;
+
+        // Always expand single points so we don't zoom
+        // directly into the ground, even when not using terrain.
+        if (destination instanceof Cartesian3) {
+            var carto = ellipsoid.cartesianToCartographic(destination);
+            var offset = CesiumMath.toRadians(0.001);
```

Comment:
I'm not convinced this is the right thing to do.  The rectangle covers a larger area closer to the equator than the poles so the camera destination will have a greater height at the equator. From experimenting, the camera height at the equator is around 500m and at the poles it's 350m.

Reply:
Yeah, I agree.  I was just moving code over from `PeliasGeocoder` and the thought just didn't occur to me.  Instead I'm going to apply a fixed offset of 500m to any single point rather than expanding it into a rectangle at all.  I'll still query the terrain at that single point, so it will still adjust properly and this will make for consistent zooming with and without terrain.